clang/include/clang/Driver/Options.td

```diff
@@ -2689,6 +2689,11 @@ def fno_sycl_link_spirv : Flag<["-"], "fno-sycl-link-spirv">,
   "when discovered in user specified objects and archives.">;
 def fsyntax_only : Flag<["-"], "fsyntax-only">,
   Flags<[NoXarchOption,CoreOption,CC1Option,FC1Option]>, Group<Action_Group>;
+defm sycl_enable_bfloat16_conversion: BoolFOption<"sycl-enable-bfloat16-conversion",
+  LangOpts<"SYCLEnableBF16Conversion">, DefaultFalse,
+  PosFlag<SetTrue, [], "Define __SYCL_BF16_CONVERSION_IS_SUPPORTED__ macro and enable">,
+  NegFlag<SetFalse, [], "Do not enable">,
+  BothFlags<[CC1Option, CoreOption], " BFloat16 conversion extension">>;
```

Comment:
```suggestion
  BothFlags<[CC1Option, CoreOption], "BFloat16 conversion extension">>;
```

Reply:
Won't we lose the space like this? I mean wouldn't be a description be like:
PosFlag: Define __SYCL_BF16_CONVERSION_IS_SUPPORTED__ macro and enableBFloat16 conversion extension
NegFlag: Do not enableBFloat16 conversion extension
?

Reply:
Oops! My suggestion was bad -- I didn't realize the marshalling options were that bad here (seems rather silly to expect users of the API to know they need a leading whitespace when the marshaller logic could insert the whitespace once rather than making every option carry extraneous whitespace). Sorry about that!